qc_schema/dev/properties/calcinfo_properties.py

```diff
@@ -0,0 +1,35 @@
+"""
+The complete list of the general computation properties.
+"""
+
+calcinfo_properties = {}
+
+calcinfo_properties["calcinfo_nbasis"] = {
```

Comment:
Do we need to duplicate `calcinfo` in the key name if it's already in the filename and the struct name?

Reply:
Keep in mind the structure is flattened. The python is just there to help build out a structure instead of writing a single 5k line schema.